src/pages/preferences/preferencesTabs/notificationTab/ClearSubscriptionsCard.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import { useMutation } from "@apollo/client";
+import styled from "@emotion/styled";
+import Button, { Variant } from "@leafygreen-ui/button";
+import Card from "@leafygreen-ui/card";
+import { Body } from "@leafygreen-ui/typography";
+import { usePreferencesAnalytics } from "analytics";
+import { useBannerDispatchContext } from "context/banners";
+import {
+  ClearMySubscriptionsMutation,
+  ClearMySubscriptionsMutationVariables,
+} from "gql/generated/types";
+import { CLEAR_MY_SUBSCRIPTIONS } from "gql/mutations";
+import { PreferencesModal } from "pages/preferences/preferencesTabs/PreferencesModal";
+
+export const ClearSubscriptionsCard: React.FC = () => {
+  const [showModal, setShowModal] = useState(false);
+  const { sendEvent } = usePreferencesAnalytics();
+  const dispatchBanner = useBannerDispatchContext();
+
+  const [clearMySubscriptions, { loading }] = useMutation<
+    ClearMySubscriptionsMutation,
+    ClearMySubscriptionsMutationVariables
+  >(CLEAR_MY_SUBSCRIPTIONS, {
+    onCompleted: (result) => {
+      setShowModal(false);
+      dispatchBanner.successBanner(
+        `Successfully cleared ${result.clearMySubscriptions} subscriptions!`
+      );
+    },
+    onError: (err) => {
+      setShowModal(false);
+      dispatchBanner.errorBanner(
+        `Error while clearing subscriptions: '${err.message}'`
+      );
+    },
+  });
+
+  return (
+    <>
+      <PreferencesCard>
+        <ContentWrapper>
+          <Body>
+            To clear all subscriptions you have made on individual Task pages.
```

Comment:
can we change this copy to "To clear all subscriptions you have made on individual Version and Task pages."?
relevant thread: https://mongodb.slack.com/archives/G013M6ULWS1/p1604515687002100